README.md

```diff
@@ -31,6 +33,20 @@ More specifically:
 
 The Dockerfiles contain instructions for how to build them individually. This is useful if you want to test just one of the docker containers.
 
+## VSCode
+
+A good option for development is to use the [VS Code Dev Containers](https://code.visualstudio.com/docs/remote/containers) extension with the [Remote - SSH](https://code.visualstudio.com/docs/remote/ssh) extension to connect to a container running on a CHAI server. This allows you to edit and rebuild `katago` easily, without rebuilding the image or editing in the command line.
```

Comment:
```suggestion
A good option for development is to use the [VS Code Dev Containers](https://code.visualstudio.com/docs/remote/containers) extension with the [Remote - SSH](https://code.visualstudio.com/docs/remote/ssh) extension to connect to a container running on a remote server. This allows you to edit and rebuild `katago` easily, without rebuilding the image or editing in the command line.
```
It's a public repo, let's make it friendly to other people trying to repro our project.